src/qt/bitcoingui.cpp

```diff
@@ -480,24 +480,16 @@ void BitcoinGUI::createMenuBar()
     connect(qApp, &QApplication::focusWindowChanged, [zoom_action] (QWindow* window) {
         zoom_action->setEnabled(window != nullptr);
     });
-#else
-    QAction* restore_action = window_menu->addAction(tr("Restore"));
-    connect(restore_action, &QAction::triggered, [] {
-        qApp->focusWindow()->showNormal();
-    });
-
-    connect(qApp, &QApplication::focusWindowChanged, [restore_action] (QWindow* window) {
-        restore_action->setEnabled(window != nullptr);
-    });
 #endif
 
     if (walletFrame) {
-        window_menu->addSeparator();
-        QAction* main_window_action = window_menu->addAction(tr("Main Window"));
-        connect(main_window_action, &QAction::triggered, [this] {
-            GUIUtil::bringToFront(this);
-        });
-
+#ifdef Q_OS_MAC		
+    window_menu->addSeparator();
```

Comment:
Fix indentation please.

Reply:
Thanks, didn't notice the indentation mistake. The tab was after _#ifdef Q_OS_MAC_ and I found it almost by mistake. Should be good now.

Reply:
p.s. is the indentation for _#ifdef Q_OS_MAC_ okay, or should i indent that as well?